extensions/lisp-mode/detective.lisp

```diff
@@ -6,7 +6,7 @@
 
 (defun %default-capture (class position)
   (let* ((line (str:split #\Space (line-string position)))
-         (name (str:replace-all ")" ""  (second line))))
+         (name (remove #\) (second line))))
```

Comment:
I realized later that it just need to fix this https://github.com/lem-project/lem/blob/main/extensions/lisp-mode/detective.lisp#L39.